composer.json

```diff
@@ -40,6 +40,7 @@
         "php-http/curl-client": "^1.7",
         "php-http/message": "^1.6",
         "predis/predis": "~1.1",
+        "stripe/stripe-php": "^6.43",
```

Comment:
Please remove this part? It's breaking versioning for the later version of `"stripe/stripe-php": "^7.1",`

Reply:
sorry, is resolved now